src/test/java/upgrade/com/atlassian/bitbucket/jenkins/internal/UpgradeTest.java

```diff
@@ -90,15 +91,15 @@ public void staticUpgradeAnalysis() throws ExecutionException, InterruptedExcept
         Map<String, Class<?>> unreleasedClasses = loadUnreleasedClasses(releaseSensitiveClasses, safelyRemovedClasses);
 
         //compare the fields found
-        compareFields(releaseSensitiveClasses, unreleasedClasses);
+        compareFields(releaseSensitiveClasses, unreleasedClasses, safelyRemovedFields);
 
         safelyRemovedClasses.keySet().forEach(staleEntry -> errorCollector.addStaleSafeListEntry(staleEntry));
```

Comment:
we need to do the same for the fields list, adding it to the error collector if it still in the list